docs/book/src/developer/providers/v1.1-to-v1.2.md

```diff
@@ -203,4 +203,6 @@ rif. [PR #6183](https://github.com/kubernetes-sigs/cluster-api/pull/6183)
   Further, if they use the label in their `manager.yaml`, it should be adjusted since v1.24 only adds the `node-role.kubernetes.io/control-plane` label.
   An example of such an accommodation can be seen in the capi-provider-aws [manager.yaml][aws-manager-yaml-a69181]
 
+- cert-manager upgraded from v1.9.1 to v1.10.1.
```

Comment:
```suggestion
- cert-manager upgraded from v1.7.2 to v1.10.1.
```
according to https://github.com/kubernetes-sigs/cluster-api/blob/release-1.1/cmd/clusterctl/client/config/cert_manager_client.go#L30

Reply:
Thanks for catching that.